src/components/create-from-builder/create-from-builder.controller.ts

```diff
@@ -383,8 +390,14 @@ export class CreateFromBuilderController implements angular.IController {
       }, (e: any) => {
         this.ctrl.error = e;
       });
-    }, (result) => {
-      this.ctrl.error = result;
+    }, (e) => {
+      if (e.data.reason === 'AlreadyExists') {
+        this.ctrl.projectNameTaken = true;
+        this.ctrl.wizardDone = false;
+        this.ctrl.currentStep = "Configuration";
```

Comment:
This does this result in any flash / flicker when it changes to the results step and back?

Reply:
Yes it does, if that is an issue I can look into wiring up `nextCallback()` and return `false' to not navigate, not sure if that'll do the trick though.

Reply:
I think that's what we do in the process-template-dialog.

Reply:
yes, and then we use eventing to navigate to Results upon success, just more overhead than what I'm doing at the moment, but if we want to avoid the flicker...

Reply:
If we do this, then there should be no need to set currentStep at all.

Reply:
Ok, then I believe I have addressed all review comments.  Should I create an issue about the proposed "+" next to the select project dropdown?